src/_data/sites/darnes.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "David Darnes",
+	"url": "https://darn.es/",
+	"description": "The personal portfolio and blog of David Darnes",
+	"twitter": "daviddarnes",
+	"authoredBy": ["daviddarnes"],
+  "business": {
+		"cta": "https://darn.es/#contact",
+    "availability": "2021-12-01"
```

Comment:
Oh interesting! I didn't know that was a supported field.
https://github.com/11ty/11ty-website/blob/d0a28dbb6d4b6f459273e71c66e63993101bae6d/src/blog/2021-02-26-espbn.md

Reply:
@pdehaan heh yea, I'm on leave for a couple of months so I thought I'd make use of it. Thanks for merging 😊